debian/firmware-nxp.install

```diff
@@ -0,0 +1,2 @@
+nxp/FwImage_IW416_SD/sdiouartiw416_combo_v0.bin	/lib/firmware/nxp
+EULA.txt /usr/share/doc/imx-firmware
```

Comment:
Wrong directory, the package name is firmware-nxp. But installing this file this way is not a good idea. If the package would be renamed the path would be needed to change by hand.

Reply:
Updated as well...